### PR TITLE
I29 - Status info now refreshes after a change is made

### PR DESCRIPTION
**Pull request by RahulR100**

### CODE REVIEW

File: server-code/Changes.js [hunk 1]
```diff
@@ -108,6 +108,17 @@ function getStatusUpdateContent(wbsNum) {
     return html;
 }
 
+/**
+* Gets the current status of the Work Package
+*
+* @return {Integer} - Status of the work package
+*/
+
+function getCurrentStatus(wbsNum) {
```
Comment (kevinyu328) [inline server-code/Changes.js:117]: Is this function currently being used? Or are you planning to do something with it in the future?

**Review by kevinyu328 — COMMENTED** [review]
Just one small question, but everything else looks great!

File: server-code/Changes.js (hunk 1 above)
Comment (RahulR100) [reply server-code/Changes.js:117]: Oop that function was just for testing and should be removed. I will correct this shortly thanks!

**Review by kevinyu328 — DISMISSED** [review]
Nice! This is a good feature to implement!

**Review by kevinyu328 — APPROVED** [review]

**Review by kevinyu328 — APPROVED** [review]
Had to re-review it after resolving conflicts.

**Pull request merged**